spaceWeatherApi/Controllers/BaseController.cs

```diff
@@ -0,0 +1,158 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using spaceWeatherApi;
+using spaceWeatherApi.DataModels;
+using spaceWeatherApi.Utils;
+using System.Globalization;
+using System.Reflection;
+
+namespace SpaceWeatherApi.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
```

Comment:
If the `BaseController` is abstract, you may not need these annotations since this seems to be a helper class more than an actual set of endpoints?